src/components/protocol_handler/test/incoming_data_handler_test.cc

```diff
@@ -56,6 +56,8 @@ class IncomingDataHandlerTest : public ::testing::Test {
     some_data2_size = 512;
     some_data = new uint8_t[some_data_size];
     some_data2 = new uint8_t[some_data2_size];
+    memset(some_data, 0xAB, some_data_size);
+    memset(some_data2, 0xAB, some_data2_size);
```

Comment:
@AByzhynar I agree with you. Unfortunately I cannot use a stack array as this will be recognized as VLA in Visual Studio and will not compile. I think it will be better to use `std::vector` as is recommended in [Scott Meyers: Effective STL](https://www.amazon.com/Effective-STL-Specific-Standard-Template/dp/0201749629?ie=UTF8&tag=stackoverfl08-20) This approach is realized in `protocol_handler_tm_test.cc`